Makefile

```
@@ -244,6 +244,20 @@ image-build:
 		./
 
+.PHONY: image-pushing
+image-pushing-presubmit:
+ifeq ($(JOB_TYPE),periodic)
+	$(MAKE) -j3 image-pushing-periodic
+else
+	$(MAKE) -j5 image-pushing-presubmit
+endif
+
+.PHONY: image-pushing-presubmit
+image-pushing-presubmit: image-push helm-chart-push kueueviz-image-push kueue-populator-image-push kueue-priority-booster-image-push
```

checkmake (0.3.2)

[warning] Target "image-pushing-presubmit" defined multiple times (lines 248 and 256).

(uniquetargets)